generator/templates/test.template.ts

```diff
@@ -69,6 +69,8 @@ contract ${contractName}_Test is Protocol${protocolVersion}TestBase {
     );
 
     diffReports('pre${contractName}', 'post${contractName}');
+
+    e2eTest(Aave${protocolVersion}${chain}.POOL);
```

Comment:
bit stupid to always ask ppl to add e2e test if we could just have it on the template 😅